libraries/DateTime/contracts/DateTime.aes

```diff
@@ -182,11 +394,10 @@ contract DateTime =
       let dt : date_time = dt{seconds_accounted_for = dt.seconds_accounted_for + state.day_in_seconds}
       p_get_day'(from + 1, to, dt)
 
-  private function get_day'(timestamp : int) : date_time =
-    let dt : date_time =  get_month'(timestamp)
-    p_get_day(dt)
+  private function rest(year : int, div : int) : int =
```

Comment:
Isn't this just `mod`?

Reply:
could you elaborate more on that, please?

Reply:
@HristiyanG there is a `rest`/`remainder`/`mod` functionality in sophia directly https://github.com/aeternity/protocol/blob/master/contracts/sophia.md#arithmetic